File: Demais/OMaior.py
# O Maior

MaiorAB = int
maior = int
a,b,c = input().split(" ")
a = int (a)
b = int (b)
c = int (c)

MaiorAB = (a+b+abs(a-b))/2
maior = (MaiorAB+c+abs(MaiorAB-c))/2

print("%d eh o maior"%maior)

File: Demais/Omaior.c
//O maior

#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

float a;
float b;
float c;

int MaiorAB ;
int maior;

int main() {

scanf("%f %f %f",&a,&b,&c);

MaiorAB = (a+b+abs(a-b))/2;
maior = (MaiorAB+c+abs(MaiorAB-c))/2;

printf("%d eh o maior",maior);

printf("\n");
}








